install/install.go

```diff
@@ -130,6 +130,11 @@ type Parameters struct {
 
 	// HelmRepository specifies the Helm repository to download Cilium Helm charts from.
 	HelmRepository string
+
+	// HelmReleaseName specifies the Helm release name for the Cilium CLI.
+	// Useful for referencing Cilium installations installed directly through Helm
+	// or overriding the Cilium CLI for install/upgrade/enable.
+	HelmReleaseName string
```

Comment:
**nitpick:** I would prefer to reorder this struct and a few others to help with increase readability and possible help others make future changes. I wasn't sure of the convention, so no changes were made.

Reply:
If done in separate commits as I think we generally accept such changes, assuming they either coming from existing contributors or are part of a larger work item (as is the case here).